docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
+
+### Step 7: Downloading Dependencies
+
+As **administrator** from PowerShell 7 (right button and “Run as Administrator”) at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_download_deps.cmd
+```
+ 
+Note: You will get a number of messages of the form below; they do not indicate errors.
+
+```
+"C:\Users\User\Neuron\nrn>pwsh -command Invoke-WebRequest -MaximumRetryCount 4 -OutFile python-3.8.exe https://www.python.org/ftp/python/3.8.2/python-3.8.2-amd64.exe   || goto :error" 
+```
+ 
+------------------------------------------------------------ 
+
+### Step 8: Installing Dependencies
+
+As **administrator** from PowerShell 7, at `C:\Users\User\Neuron\nrn\ci` run: 
+```
+.\win_install_deps.cmd
+``` 
+
+This command will take a while to complete.
+  
+------------------------------------------------------------ 
+
+### Step 9: Install Windows Subsystem for Linux:  wsl.exe --update  
+
+`wsl.exe --list --online` and 
+`wsl.exe --install Ubuntu-24.04` (or any other) 
+  
+------------------------------------------------------------ 
+
+### Step 10: (If you are installing neuron in a Virtual Machine) 
+
+Turn off your VM and at your computer as **administrator** put in PowerShell 7: 
+
+`
+Set-VMProcessor -VMName "NameOfYourVM" -ExposeVirtualizationExtensions $true
+`
+
+For example:
+
+```bash
+Set-VMProcessor -VMName "Windows 11 dev environment" -ExposeVirtualizationExtensions $true 
+```
+
+ 
+------------------------------------------------------------ 
+
+ 
+### Step 11: We need execute the “mingw64.exe” 
+So, look for "msys64", go to folder “msys64” and execute `mingw64.exe`.
+
+Via folder: open folder `C:\msys64` (probably) and double click in the executable `mingw64` 
+
+Via terminal: go to the location of the execuable `./mingw64.exe` (`C:\msys64` probably or `\\wsl.localhost\Ubuntu-24.04\mnt\c\msys64`) and execute it 
+
+```
+./mingw64.exe
+```
+ or `/mnt/c/msys64$ ./mingw64.exe `
+ 
+It will open a terminal. It is not PowerShell, it is not CMD, it is a mingw64 teminal.
+ 
+------------------------------------------------------------ 
+## How to build NEURON
+
+### Step 12: Creating the exe
+
+Now, inside this terminal (it is a mingw64 terminal, see step above) go to C:\Users\User\Neuron\nrn\ci and run 
+
+```
+./win_build_cmake.sh 
+```
+ 
+It will create an exe file `nrn-nightly-AMD64` (at `C:\Users\User\Neuron\nrn\nrn-nightly-AMD64`).
+ 
+------------------------------------------------------------ 
+
+### Step 13: Running the exe
+
+Double click at `nrn-nightly-AMD64` and it will install neuron. 
+It is a standard installation. 
+
+That means the files created at `C:\Users\User\Neuron\nrn\build\bin` will be coped to:  `C:\nrn\bin` and the "installation" will appear at `C:\nrn` and `C:\nrn-install` 
+ 
+------------------------------------------------------------ 
+
+
+### Step 14: Using and modify NEURON file... 
+You can replace any new `.dll` file at `C:\nrn\bin` and the NEURON will prevent the new feature implemented. 
```

Comment:
I'm not 100% sure what you're trying to say here, but NEURON shouldn't be preventing anything. :)

Reply:
Present